### PR TITLE
docs(components): document what start and end mean

### PR DESCRIPTION
**Pull request by brandyscarney**
in terms of padding, margin, position

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`npm run build`) was run locally and any changes were pushed
- [x] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: core/src/components/content/readme.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # ion-content
 
-Content component provides an easy to use content area with some useful methods
+The content component provides an easy to use content area with some useful methods
 to control the scrollable area. There should only be one content in a single
-view component.
+view.
+
+Content, along with all other Ionic components, can be customized to modify its padding, margin, and more using the global styles provided in the [CSS Utiltiies](/docs/layout/css-utilities) or by individually styling it using CSS and the available [CSS Custom Properties](#css-custom-properties).
```
Comment (dotNetkow) [inline core/src/components/content/readme.md]: Utiltiies

File: core/src/components/fab-button/readme.md [hunk 2]
```diff
@@ -128,25 +128,25 @@ export default Example
...
+| `--border-color`         | Border color of the button                                                                                |
+| `--border-radius`        | Border radius of the button                                                                               |
+| `--border-style`         | Border style of the button                                                                                |
+| `--border-width`         | Border width of the button                                                                                |
+| `--box-shadow`           | Box shadow of the button                                                                                  |
+| `--color`                | Text color of the button                                                                                  |
+| `--color-activated`      | Text color of the button when activated                                                                   |
+| `--color-focused`        | Text color of the button when focused                                                                     |
```
Comment (dotNetkow) [inline core/src/components/fab-button/readme.md]: More to add that could clarify Focused vs Activated? I actually don't know the difference myself....

File: core/src/components/content/readme.md (hunk 1 above)
Comment (brandyscarney) [reply core/src/components/content/readme.md]: Nice catch :joy:

File: core/src/components/fab-button/readme.md (hunk 2 above)
Comment (brandyscarney) [reply core/src/components/fab-button/readme.md]: Done! https://github.com/ionic-team/ionic/pull/18538/commits/9a1cbb3fe5eb8279dfc9bbb4af4592199e06e4cf

**Pull request merged**